app/service/app_svc.py

```diff
@@ -143,8 +144,11 @@ def __init__(self, application):
     """ PRIVATE """
 
     async def _save_configuration(self):
-        with open('conf/default.yml', 'w') as config:
-            config.write(yaml.dump(self.get_config()))
+        with open('conf/{}.yml'.format(self.config_name), 'w') as config:
+            old_config = yaml.load(config, Loader=yaml.FullLoader)
+        if self.get_config() != old_config:
+            with open('conf/default.yml', 'w') as config:
```

Comment:
should we update default.yml in this case - or the config name that we started the app with? seems like the latter..

Reply:
100% right; fixed now.